cli/endpoints/online/deploy-with-packages/custom-model/model-deployment.yml

```diff
@@ -1,5 +1,5 @@
 $schema: https://azuremlschemas.azureedge.net/latest/managedOnlineDeployment.schema.json
-name: with-package
+name: regular_deployment
 endpoint_name: hello-packages
 model: azureml:sklearn-regression@latest
 environment: azureml:sklearn-regression-env@latest
```

Comment:
Just want to double check to make sure that this was tested. When I passed the env created from the conda file i got a dependency error while deploying

Reply:
Yeap, I tested it out and looks to be working.